recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-core/base-files/base-files_%.bbappend

```diff
@@ -0,0 +1,3 @@
+do_install_append () {
+	echo "shopt -s checkwinsize && resize 1> /dev/null" >> ${D}${sysconfdir}/profile
+}
```

Comment:
Seems it should be splitted up in two lines. So shopt command would be easily commented out in case of necessity:
echo "shopt -s checkwinsize" >> ${D}${sysconfdir}/profile
echo "resize 1> /dev/null" >> ${D}${sysconfdir}/profile

Reply:
It may sense. Could you tell me how to test issue with ssh/shopt? I've tried it on my setup and it works fine.

Reply:
I had no opportunity to recheck it by myself, so al1img may share more details about it. In any case, it should be integrated as is and fixed later if the issue with ssh appears...

Reply:
Updated with suggestions.